packages/ninja.rb

```diff
@@ -3,23 +3,23 @@
 class Ninja < Package
   description 'a small build system with a focus on speed'
   homepage 'https://ninja-build.org'
-  version '1.11.1'
+  version '1.11.1-1'
   license 'GPL-2'
   compatibility 'all'
-  source_url 'https://github.com/ninja-build/ninja/archive/v1.11.1.tar.gz'
-  source_sha256 '31747ae633213f1eda3842686f83c2aa1412e0f5691d1c14dbbcc67fe7400cea'
+  source_url 'https://github.com/ninja-build/ninja.git'
+  git_hashtag 'v1.11.1'
```

Comment:
Use `@_ver`?

Reply:
done.

Reply:
I'm not seeing a change.  Did I miss it?

Reply:
one git push later...